lib/Browser.js

```diff
@@ -129,14 +132,16 @@ class Browser extends EventEmitter {
 
   disconnect() {
     this._connection.dispose();
+    this.emit(Browser.Events.Disconnected, 'connection disconnected');
```

Comment:
no need to emit here: this will result in disconnected event dispatched twice (one from here, another from the connection been closed)